github-repo-manager/src/main/java/com/axonivy/github/GitHubFileDetector.java

```diff
@@ -0,0 +1,110 @@
+package com.axonivy.github;
+
+import java.io.IOException;
+import java.text.MessageFormat;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHRepository;
+
+public abstract class GitHubFileDetector {
```

Comment:
In this repo the standard indent is two spaces. Please replace all /t to two spaces "  ".